### DISCUSSION TITLE
Define the number of words for SRT files

### DISCUSSION THREAD

**Discussion by vallbrd** [chidiwilliams/buzz#534] [Ideas]
Having the option to set the maximum number of characters per sentence for SRT files? It would be incredibly convenient to directly import the SRT file into video editing software and have subtitles that perfectly fit the length of my content.

Adobe Premiere Pro has something similar. They have a speech-to-text feature that allows users to set a character limit per line when creating captions. You can check it out here: https://helpx.adobe.com/premiere-pro/using/speech-to-text.html/#create-captions

I'm confident that this feature would be a huge asset for video editors and other professionals who use application.

**Comment by surdophobe** [comment]
Absolutely!  The DMCP recommends 32 characters per line but that's partially due to limitations in captioning equipment.  More recent sources vary but recommend 42-49 depending on the source. As a closed captioning user having lines at a limited length is very important for readability.
